design/ux/Obscuro_Gateway.md

```diff
@@ -129,7 +129,39 @@ Alice -> OG: All further Obscuro interactions will be to\nhttps://gateway.obscur
 The onboarding should be done in 3 clicks.
 1. The user goes to a website (like "obscuro.network"), where she clicks "Join Obscuro". This will add a network to their wallet.
 2. User connects the wallet to the page.
-3. In the wallet popup, the user has to sign over a message: "Register $UserId for $ACCT"
+3. In the wallet popup, the user has to sign over EIP-712 formatted message.
+
+Format of EIP-712 message used for signing viewing keys is:
+
+```
+types: {
+      EIP712Domain: [
+        { name: "name", type: "string" },
+        { name: "version", type: "string" },
+        { name: "chainId", type: "uint256" },
+        { name: "verifyingContract", type: "address" },
```

Comment:
what will be the "verifyingContract"?

Reply:
From: https://eips.ethereum.org/EIPS/eip-712
`address verifyingContract` the address of the contract that will verify the signature. The user-agent may do contract specific phishing prevention.

But verifying contract can be removed since we don't use it in that way. WIll remove it.